ux-design/prototype/assets/scripts.js

```diff
@@ -1,26 +1,26 @@
-;
-;
-/*! jquery-ui-1.9.1.custom.min.js */ 
-;
-;
+
```

Comment:
Any idea what happened with this file? GitHub seems to think the entire thing was rewritten, but I don't see any changes and I have no idea why it's doing that haha

Reply:
lol i dont know, I haven't  really modified it. probably when the files compile/run they automatically update some scripts